scripts/start-finalExec

```diff
@@ -294,7 +294,10 @@ elif [[ $SERVER =~ run.sh ]]; then
     echo "SETUP_ONLY: bash ${SERVER}"
     exit
   fi
-  exec mc-server-runner "${mcServerRunnerArgs[@]}" --shell bash "${SERVER}"
+  if isTrue "${DEBUG_EXEC}"; then
+    set -x
+  fi
+  exec mc-server-runner "${mcServerRunnerArgs[@]}" --shell bash "${SERVER}" "${EXTRA_ARGS}"
```

Comment:
and just saw your follow up. Removing the double quotes around the car should help it here. 

Sorry if I merged your PR too soon. Will need a new one for the tweak.